bpmn2stamp-converter/pom.xml

```diff
@@ -250,7 +250,7 @@
                 <executions>
                     <execution>
                         <id>main</id>
-                        <phase>compile</phase>
+                        <phase>process-classes</phase>
```

Comment:
@grigobog NOTE this change -- this is not needed to fix this commit, but don't understand motivation why there was "compile" value instead. So I just wanted to warn you I changed it.